From 1efd319579c8f34974d654b0a5ac5144424e693e Mon Sep 17 00:00:00 2001
From: Glauber Costa <glommer@redhat.com>
Date: Fri, 13 Feb 2009 13:00:31 -0500
Subject: [PATCH] use KVM_UPSTREAM for ppc.

ppc should compile with upstream qemu code, so, put these
defines in ppc specific code that references kvm functions.

We don put them in config.h in this case, since there are
files (like vl.c) that includes both kvm.h and qemu-kvm.h,
and would break compilation.

Signed-off-by: Glauber Costa <glommer@redhat.com>
---
 qemu/hw/ppc440.c         |    1 +
 qemu/hw/ppc440_bamboo.c  |    1 +
 qemu/target-ppc/helper.c |    1 +
 3 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/qemu/hw/ppc440.c b/qemu/hw/ppc440.c
index 00d82e4..164c326 100644
--- a/qemu/hw/ppc440.c
+++ b/qemu/hw/ppc440.c
@@ -18,6 +18,7 @@
 #include "ppc440.h"
 #include "ppc405.h"
 #include "sysemu.h"
+#define KVM_UPSTREAM
 #include "kvm.h"
 
 #define PPC440EP_PCI_CONFIG     0xeec00000
diff --git a/qemu/hw/ppc440_bamboo.c b/qemu/hw/ppc440_bamboo.c
index fbd447c..60ddaf4 100644
--- a/qemu/hw/ppc440_bamboo.c
+++ b/qemu/hw/ppc440_bamboo.c
@@ -21,6 +21,7 @@
 #include "boards.h"
 #include "sysemu.h"
 #include "ppc440.h"
+#define KVM_UPSTREAM
 #include "kvm.h"
 #include "kvm_ppc.h"
 #include "device_tree.h"
diff --git a/qemu/target-ppc/helper.c b/qemu/target-ppc/helper.c
index fc62a63..d49acaf 100644
--- a/qemu/target-ppc/helper.c
+++ b/qemu/target-ppc/helper.c
@@ -29,6 +29,7 @@
 #include "exec-all.h"
 #include "helper_regs.h"
 #include "qemu-common.h"
+#define KVM_UPSTREAM
 #include "kvm.h"
 
 //#define DEBUG_MMU
-- 
1.5.6.5

